service/src/test/java/uk/gov/hmcts/reform/fpl/model/JudgeTest.java

```diff
@@ -43,6 +44,34 @@ void shouldReturnAllocatedJudgeLastNameWhenMagistratesIsNotSelected() {
         assertThat(lastName).isEqualTo("Judge Last Name");
     }
 
+    @Test
+    void shouldReturnTrueWhenAllocatedJudgeAndJudgeAndLegalAdvisorAreEqual() {
```

Comment:
shouldReturnTrueWhenJudgesAreEqual() ?

Bit easier to read, don't like the JudgeAndJudgeAndLegalAdvisor